pkg/directorysync/client.go

```diff
@@ -447,6 +447,9 @@ type ListDirectoriesOpts struct {
 	// Searchable text for a Directory. Can be empty.
 	Search string
 
+	// Organization ID of a Directory. Can be empty.
+	OrganizationID string
```

Comment:
We need to also read this value from the options and add it to the URL's query, similar to what we do with the other parameters [here](https://github.com/workos-inc/workos-go/blob/0ba82c57e07e0cab2a6e2dffdb9d99a95a1a798e/pkg/directorysync/client.go#L497).

Reply:
Done, I believe.